ewwep.py

```diff
@@ -1763,7 +1779,14 @@ def apply_combat_mods(user_data = None, desired_type = None, target = None):
 	user_isslimecorp = user_data.life_state in [ewcfg.life_state_lucky, ewcfg.life_state_executive]
 
 	# hunger drain
-	user_data.hunger += ewcfg.hunger_pershot * ewutils.hunger_cost_mod(user_data.slimelevel)
+	if not sandbag_mode:
+		user_data.hunger += ewcfg.hunger_pershot * ewutils.hunger_cost_mod(user_data.slimelevel)
+		
+	if sandbag_mode:
+		backfire = False
+		miss = True
```

Comment:
if they always miss, won't players be unable to see how much damage their weapon does?